admin/views/partials/_body_start.ejs

```diff
@@ -1,9 +1,9 @@
+<!-- Global Site Tag (gtag.js) - Google Analytics -->
+<script async src='https://www.googletagmanager.com/gtag/js?id=<%= googleTrackingId %>'></script>
 <script>
-    var googleTrackingId = '<%= googleTrackingId %>';
-    if (googleTrackingId) {
-      window.ga=window.ga||function(){(ga.q=ga.q||[]).push(arguments)};ga.l=+new Date;
-      ga('create', googleTrackingId, 'auto');
-      ga('send', 'pageview');
-    }
+  window.dataLayer = window.dataLayer || [];
```

Comment:
you need the IF block to deal with situations where we have not added a tracking code and GA should not be enabled

Reply:
@GuyHarwood very true, it is updated